Conduit-Probe.md

```diff
@@ -1,12 +1,12 @@
+# Conduit Probe
 ![](http://loenwind.info/eio/Conduit_Probe.png)
 
-Conduit Probe
-
 The Conduit Probe has 2 modes:
 
-* In probe mode you can right-click a [[Conduit|conduit]] to display details about its network.
-* In copy/paste mode you can shift-right-click to copy a conduit connection's settings and right-click to paste them.
+* In probe mode ![](https://github.com/SleepyTrousers/EnderIO/blob/master/enderio-base/src/main/resources/assets/enderio/textures/items/item_conduit_probe_probe.png) you can right-click a [[Conduit|List of Conduits]] to display details about its network.
```

Comment:
Can you add somewhere that when clikcing an item conduit with the probe in one hand, it will run a simulation on transporting the item in the other hand? Maybe in a more understandable way than what I just wrote? thx!!